docs/project-overview.md

```diff
@@ -0,0 +1,83 @@
+
+## Reference Implementations
+### Tokens
+In order that the general logic within the components performs the correct set of operations to provide support for the maximum set of deployments, many real world attestation solutions and potential extrapolations of their use have been considered. Developing understanding and modelling the necessary flows for their verification process has driven the Veraison design.
+To ensure that the code backs up the design, reference support for multiple tokens will be developed in the project. The current targets for this support are:
+* EAT Tokens, with the ARM PSA Profile being a specific test set. 
+* DICE Tokens. NB: the current delivery format for the 'token', being the full DICE Certificate chain is still under consultation. If you're planning any such protocol implementations, we would be very interested in discussing them.
+
+### Deployments
+To ensure that the core components are structured correctly for use as a service, the project will build reference deployments that can perform token verifications. To help ensure abstractions for the components over deployment environments are sufficiently general, multiple reference deployments will be built. They are:
+* A deployment for self hosting
+* A deployment targeting a hyperscaler PAAS environments
+Further details on the above will be available within specific documentation on the reference deployment. (ToDo) 
+
+## Scope 
+The following is the intended scope of the Veraison Project. This is being listed to help others understand the intended functionality. If you're considering adopting or contributing to Veraison and you think there's something missing for the core of an Attestation Verification service, let us know via the Zulip channel. The following list should not be treated as a backlog, work items will be recorded elsewhere. 
+### Scope - Verification
+* An API to which an Attestation token, containing evidence claims, can be submitted. The API can either produce a simple boolean 'approved' response or a complex set of Attestation Results (verified claim summary). The results will be in a (signed) JSON format.
+* The implementation of the API will be extensible to new formats & scenarios (e.g. encrypted tokens) by plugins
+* The Appraisal Policy for the submitted evidence can be specified as a policy using the OPA (Open Policy Agent) language 'rego'. A plugin point for code, to support complex appraisal, will also be available.
+* The Policy can access Reference Values or Endorsements required to support the Appraisal by specifying queries
+* The API will record basic metrics for integration into deployment operational schemes
+* In addition to the direct Verification of the supplied evidence, further policy can be deployed to add 'Derived Claims' to the Attestation results.
```

Comment:
since we haven't fleshed this out yet, I'd mark it as "work in progress" or similar

Reply:
The section is labelled as intended scope so I think we're OK to not treat this too differently